sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -913,15 +986,12 @@ class SqlRuleFilter(object):
     :type sql_expression: str
     :param parameters: Sets the value of the sql expression parameters if any.
     :type parameters: dict[str, Union[str, int, float, bool, datetime, timedelta]]
-    :param requires_preprocessing: Value that indicates whether the rule
-     filter requires preprocessing. Default value: True .
-    :type requires_preprocessing: bool
     """
-    def __init__(self, sql_expression=None, parameters=None, requires_preprocessing=True):
-        # type: (Optional[str], Optional[Dict[str, Union[str, int, float, bool, datetime, timedelta]]], bool) -> None
+    def __init__(self, sql_expression=None, parameters=None):
+        # type: (Optional[str], Optional[Dict[str, Union[str, int, float, bool, datetime, timedelta]]]) -> None
         self.sql_expression = sql_expression
         self.parameters = parameters
-        self.requires_preprocessing = requires_preprocessing
+        self.requires_preprocessing = None
```

Comment:
would setting the value to `None` affect functionality? because previously by default the value is `True` and I see it is used in `_to_internal_entity`.

Reply:
Great catch.  I had intended to maintain existing functionality with the params we yanked, must have elided this one.